src/main/java/org/trigon/hardware/misc/servo/io/RealServoIO.java

```diff
@@ -0,0 +1,56 @@
+package org.trigon.hardware.misc.servo.io;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.Servo;
+import org.trigon.hardware.misc.servo.ServoIO;
+import org.trigon.hardware.misc.servo.ServoInputsAutoLogged;
+
+public class RealServoIO extends ServoIO {
+    private Rotation2d
+            maxServoAngle = Rotation2d.fromDegrees(180),
+            minServoAngle = Rotation2d.fromDegrees(0);
```

Comment:
1. Non final above final
2. Only a 180 servo has angle limits, a 360 one means it doesn't have limits